src/DocumentScope.zig

```diff
@@ -726,7 +726,10 @@ noinline fn walkContainerDecl(
                 const name_token = tree.fullVarDecl(decl).?.ast.mut_token + 1;
                 if (name_token >= tree.tokens.len) continue;
 
-                const name = offsets.tokenToSlice(tree, name_token);
+                var name = offsets.tokenToSlice(tree, name_token);
+                if (name.len > 1 and name[0] == '@' and name[1] == '"' and name[name.len - 1] == '"') {
+                    name = name[2 .. name.len - 1];
+                }
```

Comment:
There are many more places where this check is necessary. See #1574

Reply:
Will rebase on top of it